incubator/kube2iam/templates/daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      hostNetwork: true
+      containers:
+      - name: kube2iam
+        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        args:
+        {{- if .Values.host.iptables }}
+        - --host-ip={{ default "$(HOST_IP)" .Values.host.ip }}
+        {{- end }}
+        - --iptables={{ default false .Values.host.iptables }}
+        {{- if .Values.host.interface }}
```

Comment:
You can safely remove this `if` around `--host-interface`. kube2iam defaults to `docker0` if this argument isn't given, so there's no risk of changing behavior if `.Values.host.interface` isn't defined.

Then (IMHO) move it to the top of the args list for clarity.